subscription.go

```diff
@@ -276,8 +276,15 @@ func (s *Subscription) Stats(ctx context.Context) (*ua.SubscriptionDiagnosticsDa
 		return nil, errors.Errorf("empty SubscriptionDiagnostics for sub=%d", s.SubscriptionID)
 	}
 
-	for _, eo := range v.Value().([]*ua.ExtensionObject) {
-		stat := eo.Value.(*ua.SubscriptionDiagnosticsDataType)
+	extensions, ok := v.Value().([]*ua.ExtensionObject)
```

Comment:
* Please use `eos` instead of `extensions`
* `return nil, errors.Errorf("invalid type for SubscriptionDiagnosticsArray. Want []*ua.ExtensionObject. subID=%d nodeID=%s type=%T", s.SubscriptionID, node.String(), v.Value())`
* Please add a blank line between the `if` block and the `for` block

Reply:
Thanks for the review! Those should be fixed now.